source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0321
+
+<b>Functional Description</b>
+
+This Data Item can take the value '0' (zero) in the following cases: <br/>a. a document number is missing (i.e. it shall not be filled in with a dummy number); <br/>b. the length of a document number exceeds the allowed 70 characters (i.e. it shall not be truncated).<br/>A missing document reference number (due to the above or any other case) is not a valid reason for the rejection of this message.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0332
+
+<b>Functional Description</b>
+
+IF &lt;Container indicator&gt; is NOT PRESENT then data group &lt;TRANSPORT EQUIPMENT&gt; shall NOT be PRESENT, too. &lt;Container indicator&gt; functions as the governing data item for data group &lt;TRANSPORT EQUIPMENT&gt;.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0360
+
+<b>Functional Description</b>
+
+IF discrepancies have been found in one or more Data Groups or Data Items     <br/>OR<br/>a new data element has been found during the control<br/>THEN the D.G. / D.I.= "R" and is used to report these discrepancies<br/>ELSE the D.G. / D.I. = "N". 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0367
+
+<b>Functional Description</b>
+
+IF the message is a Negative CD003C/CD038C/CD115C/CD165C<br/>THEN this Data Item is Required <br/>ELSE this Data Item is not used.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0414
+
+<b>Functional Description</b>
+
+In case of excise goods where &lt;CONSIGNMENT.HOUSE CONSIGNMENT.CONSIGNMENT ITEM.SUPPORTING<br/>DOCUMENT.Type&gt; is EQUAL to 'C651 -AAD -Administrative Accompanying Document (EMCS)', the Administrative Reference Code (ARC number) shall be recorded in this field;<br/>In case of excise goods where &lt;CONSIGNMENT.HOUSE CONSIGNMENT.CONSIGNMENT ITEM.SUPPORTING<br/>DOCUMENT.Type&gt; is EQUAL to 'C658 -FAD -Fallback e-AD (EMCS)', the national Fallback registration number shall be recorded in this field.<br/><br/>
+
+<b>Technical Description</b>
+
+N/A
+
+##G0424
+
+<b>Functional Description</b>
+
+In case of excise goods where &lt;CONSIGNMENT.HOUSE CONSIGNMENT.CONSIGNMENT ITEM.ADDITIONAL REFERENCE.Type&gt; is EQUAL to 'C651 -AAD -Administrative Accompanying Document (EMCS)', the<br/>Administrative Reference Code (ARC number) shall be recorded in this field;<br/>In case of excise goods where &lt;CONSIGNMENT.HOUSE CONSIGNMENT.CONSIGNMENT ITEM.ADDITIONAL REFERENCE Type&gt; is EQUAL to 'C658C -FAD -Fallback e-AD (EMCS)’, the national Fallback registration number shall be recorded in this field.<br/>
+
+<b>Technical Description</b>
+
+N/A
+
+##G0500
+
+<b>Functional Description</b>
+
+The exact content of the CL326 (QualifierOfTheIdentification) is defined nationally, considering -for example -that only in some NAs the value 'T' must only be used in case “House number” and “Postcode” or only “Postcode” define an exact and unique location.
+
```

Comment:
The text considering -for example - is a mess. Suggest splitting sentence by a adding a full stop after `nationally` and starting second sentence with `For example, in only some NAs the...`